cmake/externals/base64project.cmake

```diff
@@ -0,0 +1,14 @@
+FetchContent_Declare(
+  base64project
+  GIT_REPOSITORY https://github.com/ReneNyffenegger/cpp-base64.git
```

Comment:
base64 is a very simple encoding, should we remove the dependency on this personal project?

Reply:
I can implement the algorithm inside ort-custom.